staging/src/k8s.io/api/batch/v2alpha1/types.go

```diff
@@ -94,6 +94,11 @@ type CronJobSpec struct {
 	// The schedule in Cron format, see https://en.wikipedia.org/wiki/Cron.
 	Schedule string `json:"schedule" protobuf:"bytes,1,opt,name=schedule"`
 
+	// The time zone to be used when scheduling the job given the schedule specified above.
+	// Valid options include anything in the IANA Time Zone database.
+	// +optional
+	TimeZone string `json:"timeZone" protobuf:"bytes,8,opt,name=timeZone"`
```

Comment:
Now that https://github.com/kubernetes/kubernetes/pull/41901 is merged, mind adding it also to `batch/v1beta1`. It's part of the beta reqs.